src/lib.rs

```diff
@@ -95,3 +95,9 @@ mod utils;
 
 #[cfg(test)]
 mod testbase;
+
+#[cfg(doctest)]
+use doc_comment::doctest;
+
+#[cfg(doctest)]
+doctest!("../README.md", readme);
```

Comment:
Let's do it like this:

```suggestion
#[cfg(doctest)]
doc_comment::doctest!("../README.md", readme);
```

Reply:
Done and squashed it into the doc-comment commit.